app/main.py

```diff
@@ -1 +1,43 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str, appetite: int, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self):
+        if self.is_hungry is True:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+        else:
+            self.appetite = 0
```

Comment:
You shouldn't change the appetite of an animal when it's not hungry. This integer just shows a number of food points that are required to feed an animal IF it's hungry. If it isn't hungry no changes are required.